### PR TITLE
Add MapML 'Authoring' section

### PR DESCRIPTION
**Pull request by Malvoz**
See motivation in https://github.com/Maps4HTML/MapML-Proposal/issues/50.

@prushforth anything to add/change? let me know!

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -3167,6 +3167,16 @@ <h5>The <code>&lt;<dfn id="the-coordinates-element">coordinates</dfn>&gt;</code>
         </p>
         </section>
         </section>
+        
+        <section class="informative">
+        <h3>Authoring</h3>
+        <p>
+        While MapML is human-readable and can be written by hand,
+        authors typically will use conversion programs
+        and other specialized software tools to generate MapML.
```
Comment (Malvoz) [inline spec/index.html:3176]: I believe "authors" in this context can mean both the application author and map data provider. Should this be clarified? If so, how exactly?

Edit: I think it'd be a good idea to state this early in the document (i.e. in the abstract), similar to MathML:

https://w3c.github.io/mathml/#abstract:

> While MathML is human-readable, authors typically will use equation editors, conversion programs, and other specialized software tools to generate MathML. Several versions of such MathML tools exist, both freely available software and commercial products, and more are under development.

Comment (prushforth) [reply spec/index.html:3176]: Looks good to me.  The equivalent in mapping are servers like GeoServer and MapServer, which serve images and documents according to standard APIs, like WMS, WFS etc.  I thought I had mentioned them somewhere, but I agree it should be up front.  Entering coordinates by hand is daunting for simple image maps; much more so for actual geographic features and so on.

**Comment by Malvoz** [comment]
Moved the MapML Example (image) to the new Authoring section in https://github.com/Maps4HTML/MapML/pull/202/commits/92950e292d214cc4a6f530d0652a7df9ce95f24f for consistency with the [Web Maps Authoring](https://maps4html.org/MapML/spec/#authoring) section.

File: spec/index.html [hunk 2]
```diff
@@ -3167,29 +3167,44 @@ <h5>The <code>&lt;<dfn id="the-coordinates-element">coordinates</dfn>&gt;</code>
...
+            <figure>
+              <a href="examples/mapml.png" style="cursor:zoom-in;">
+                <img src="examples/mapml.png" width="3240" height="2016" loading="lazy" alt="Origins, orientations and axis names of tcrs, pcrs, gcrs, tilematrix, map, and tile coordinate systems">
+              </a>
+              <figcaption>
+              Prototype MapML services are available,
+              representing
+              <a href="https://geogratis.gc.ca/mapml/en/osmtile/cbmt/">tiled</a>
+              and
+              <a href="https://geogratis.gc.ca/api/beta/vectors/canvec/50k/features/?entry-type=full&xmin=-75.6088650226593&ymin=45.39715420088217&xmax=-75.59665560722351&ymax=45.401975540403974&zoom=17&projection=WGS84&alt=xml">
```
Comment (Malvoz) [inline spec/index.html]: This link (https://geogratis.gc.ca/api/beta/vectors/canvec/50k/features/?entry-type=full&xmin=-75.6088650226593&ymin=45.39715420088217&xmax=-75.59665560722351&ymax=45.401975540403974&zoom=17&projection=WGS84&alt=xml) does not work atm. @prushforth should it be replaced with another URL?

Comment (prushforth) [reply spec/index.html]: Eventually we'll have a replacement, but that link is likely gone forever. Perhaps change the sentence for now.

Comment (Malvoz) [reply spec/index.html]: I removed the link. Please suggest something if you want the sentence changed, otherwise I think we can merge this.

**Pull request merged**